nemo/collections/asr/parts/preprocessing/perturb.py

```diff
@@ -685,6 +689,7 @@ def __init__(
         self._max_additions = max_additions
         self._max_duration = max_duration
         if bg_noise_manifest_paths:
```

Comment:
I think `if something is not None` is better for this kind of argument check since we can avoid bg_noise_manifest_paths=0 passing the if statement

Reply:
The current way is to test all of `bg_noise_manifest_paths==None`, `bg_noise_manifest_paths==[]` and `bg_noise_manifest_paths==0` conditions, so the current way should be fine